style.css

```
@@ -1 +1,246 @@
+}
+
+.cakes {
+    width: 100px;
+    height: 75px;
+    margin: 2px;
+    border: 2px solid rgb(255, 149, 193);
+}
+
+.giantCake, .cupcakes {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: rgb(255, 149, 193);
+    margin-top: 20px;
+    padding-top: 40px;
+    padding-bottom: 40px;
+}
+
+.cupcakes {
+    padding: 40px;
+    flex-flow: row wrap;
+    background-color:rgb(243, 236, 141);
+}
+
+footer {
+    background-color: rgb(33, 248, 248);
+    margin-top: 20px;
+    margin-bottom: 20px;
+}
+
+.socialMedia {
+    display: flex;
+    flex-flow: column;
+    width: 20px;
+    height: 20px;
+    padding: 5px;
+    padding-left: 12px;
+    filter: sepia(200%) hue-rotate(280deg) saturate(250%);
+}
+
+.contact {
+    font-size: 12px;
+    text-align: center;
+}
+
+@media (min-width: 420px) {
+    .logo {
+        width: 200px;
+        height: auto;
+    }
+    h1 {
+        font-size: 25px;
+    }
+    .giantCupcake {
+        width: 350px;
+        height: 450px;
+    }
+    .cakes {
+        width: 150px;
+        height: 100px;
+    }
+    .intro, p {
+        font-size: 20px;
+    }
+    .socialMedia {
+        width: 40px;
+        height: 40px;
+        padding-left: 20px;
+        padding-bottom: 10px;
+    }
+    .contact {
+        font-size: 20px;
+    }
+
+@media (min-width: 532px) {
+    .logo {
+        width: 200px;
+        height: auto;
+    }
+    h1 {
+        font-size: 25px;
+    }
+    .giantCupcake {
+        width: 350px;
+        height: 450px;
+    }
+    .cakes {
+        width: 210px;
+        height: 140px;
+    }
+    .intro, p {
+        font-size: 20px;
+    }
+    .socialMedia {
+        width: 40px;
+        height: 40px;
+        padding-left: 20px;
+        padding-bottom: 10px;
+    }
+    .contact {
+        font-size: 20px;
+    }
+
+@media (min-width: 752px) {
+    .logo {
+        width: 200px;
+        height: auto;
+    }
+    h1 {
+        font-size: 40px;
+    }
+    .giantCupcake {
+        width: 550px;
+        height: 750px;
+    }
+    .cakes {
+        width: 320px;
+        height: 200px;
+    }
+    .intro, p {
+        font-size: 25px;
+    }
+    .socialMedia {
+        width: 60px;
+        height: 60px;
+        padding-left: 20px;
+        padding-bottom: 10px;
+    }
+    .contact {
+        font-size: 25px;
+    }
+
+@media (min-width: 1025px) {
+    .logo {
+        width: 200px;
+        height: auto;
+    }
+    h1 {
+        font-size: 40px;
+    }
+    .giantCupcake {
+        width: 550px;
+        height: 750px;
+    }
+    .cakes {
+        flex-direction: row;
+        width: 450px;
+        height: 275px;
+    }
+    .intro, p {
+        font-size: 25px;
+    }
+    .socialMedia {
+        display: inline;
+        transform: translateX(50%);
+        width: 60px;
+        height: 60px;
+        padding-top: 20px;
+        padding-left: 20px;
+        padding-bottom: 10px;
+    }
+    .contact {
+        font-size: 25px;
+    }
```

```suggestion
    }
   } 
```
you missed a closing bracket for @media query